### ISSUE TITLE
Create custom base ui theme

### ISSUE THREAD

**Issue by tobiasdiez** [koppor/jabref#309]
Currently, we use the built-in JavaFX theme "Modena" as the base and overwrite certain styles. In some cases this is a bit problematic and it would be better to copy modena.css and make our changes directly to this file.
The new base UI css file can be registered using https://docs.oracle.com/javase/8/javafx/api/javafx/application/Application.html#setUserAgentStylesheet-java.lang.String-

**Comment by Siedlerchr** [comment]
I guess with our DarkTheme and the LightTheme this issue is now obsolete.

**Comment by tobiasdiez** [comment]
Nope, the `Base.css` is applied on top of `Modena`. Since we have many changes, it would be worthwhile to completely replace Modena by our base css.
